Source/WebCore/loader/EmptyClients.cpp

```diff
@@ -505,7 +507,7 @@ class EmptyStorageNamespaceProvider final : public StorageNamespaceProvider {
         PAL::SessionID m_sessionID;
     };
 
-    Ref<StorageNamespace> createSessionStorageNamespace(Page&, unsigned) final;
+    RefPtr<StorageNamespace> maybeCreateSessionStorageNamespace(SecurityOrigin&, Page&, unsigned, bool) final;
```

Comment:
The naming and parameter are confusing. I think either we use two functions createSessionStorageNamespace and existingSessionStorageNamespace, or you change this to sessionStorageNamespace(SecurityOrigin&, Page&, ShouldCreateIfNotExists) where ShouldCreateIfNotExists is a enum class.

Reply:
Initially I created two separate functions as you suggest, but that seemed more complicated, so I re-wrote the patch and renamed the function. I'll make this change.

Reply:
If you think two functions are too complicated you can use enum class like
`enum class ShouldCreateIfNotExists : bool { No, Yes };`
We usually do this for readability nowadays